plugins/terraform/src/components/TerraformLatestRun/TerraformLatestRun.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { CircularProgress, Grid } from '@material-ui/core';
+import { ResponseErrorPanel } from '@backstage/core-components';
+import { TerraformLatestRunContent } from '../TerraformLatestRunContent';
+import { useRuns } from '../../hooks';
+import { Run } from '../../hooks/types';
+import {
+  isTerraformAvailable,
+  TERRAFORM_WORKSPACE_ANNOTATION,
+  TERRAFORM_WORKSPACE_ORGANIZATION,
+} from '../../annotations';
+import { MissingAnnotationEmptyState, useEntity } from '@backstage/plugin-catalog-react';
+
+
+export const TerraformLatestRun = () => {
+
+  const { entity } = useEntity();
+
+  const { annotations } = entity.metadata;
+  const organization = annotations?.[TERRAFORM_WORKSPACE_ORGANIZATION] ?? 'undefined';
+  const workspaceName = annotations?.[TERRAFORM_WORKSPACE_ANNOTATION] ?? 'undefined';
```

Comment:
I don't think we should ever be setting a value to a string `'undefined'`, will cause a whole load of confusion. What's the reason for this? Is it because `useRuns` requires 2 string args?

Reply:
That's correct.  Without the assignation, we get the following error:

![undefined optional string error](image.png)

Reply:
That's correct.  Without this, we get a compiler error at useRuns: _"Argument of type 'string | undefined' is not assignable to parameter of type 'string'.  Type 'unassigned' is not assignable to type 'string'."_